src/ml/neural_net/mps_layer_helper.h

```diff
@@ -11,20 +11,30 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class TCMPSConvolutionWeights;
+@class TCMPSInstanceNormDataLoader;
+
+@interface MPSCNNFullyConnectedNode(TCMPSLayerProperty)
+@property (nonatomic, retain) TCMPSConvolutionWeights *weights API_AVAILABLE(macosx(10.14));
```

Comment:
Nit: in ARC projects like ours, use `strong` instead of `retain`. Although I think they are functionally equivalent

Reply:
Huh, where is this property implemented? And where is it used?

Reply:
It's currently not used anywhere. It'll be used in the future implementations of `Drawing Classifier` and `Sound Classifier`. Since we'd already completed the helper methods for Convolution and Instance Normalization, it only felt apt to add the property to the Fully Connected helper method.

Reply:
Fixed to use strong instead of retain

Reply:
IIRC, properties in categories cannot be auto-synthesized, since the category cannot modify the in-memory layout of the object. So defining the property here without providing implementations (that don’t rely on introducing new state) is just creating an interface that will yield an error if anybody tries to use it. We should either add an implementation or remove the interface until we need it.

(BTW, I should have mentioned that with ARC, `strong` is the default anyway...)

Reply:
Hopefully, I kinda fixed it with `@dynamic`?